### ISSUE TITLE
Ability to run multiple launch events sequentially instead of in parallel

### ISSUE THREAD

**Issue by ChaosCA** [Microsoft/vscode#57760]
Currently you can use compound launch configurations to trigger to launch's together.

I'm looking for the ability to trigger one launch, then after that one completes trigger another.

This is useful for launching a process, and then attaching to a process that the first one launched and then exited from. Right now it's also not possible to run an attach configuration with the `postDebugTask` because there's not way for a task to run a `launch.json` event.

A possible example is below:
```
{
    "version": "0.2.0",
    "configurations": [
        {
            "type": "node",
            "request": "launch",
            "name": "Server",
            "program": "${workspaceFolder}/server.js",
            "cwd": "${workspaceFolder}"
        },
        {
            "type": "node",
            "request": "attach",
            "name": "Debug process",
            "processName": "spawnedprocess.exe",
        }
    ],
    "compounds": [
        {
            "name": "Run the server and then attach to a process it spawns",
            "configurations": ["Server", "Debug process"],
            "order": "sequential"
        }
    ]
}
```

**Comment by almir-campos** [comment]
> We do not plan to support this.

It's Nov 2021, and it seems that we still don't have this feature. Could someone elaborate a little bit more about the reasons why Microsoft doesn't have plans for that, please?

Please, don't take my question as a demand from me, or something rude like that. I'm genuinely interested in to know what are the restraints (technical, management, resources, etc), and whether it's something I could wait for in the near future or just forget it.

Maybe there's another solution in the community that could be good enough for most people. For example, there is an Intellij-IDEA extension (Multirun) that can do this task. It's far from ideal, but sometimes it's good enough. Another possibility is to use macros; would it work? I've not tried this, yet. And so on.

I just ask you to share something with us and see what happens. Maybe the community bring some ideas. Hopefully. :)

_Ps: I empathize with you, guys. We're all developers and we (should) know that "simple" things are never as simple as one can initially think. They become nightmares very often..._

**Comment by starball5** [comment]
Duplicate of the later-posted and non-shut-down #71885?
